src/game.cpp

```diff
@@ -238,8 +243,39 @@ void Game::start() {
 	text.setText(u"OpenAWE - v0.0.1");
 	text.show();
 
+	_window->hideMouseCursor();
+	// Allow locking and unlocking the mouse
+	EventMan.setActionCallback({ kLockMouse }, [&](Events::Event event){_window->hideMouseCursor();});
+	EventMan.setActionCallback({ kUnlockMouse }, [&](Events::Event event){_window->showMouseCursor();});
+	EventMan.addBinding(kLockMouse, Events::kMouseLeft);
+	EventMan.addBinding(kUnlockMouse, Events::kKeyEscape);
```

Comment:
If you want to keep the lock/unlock mouse mechanic on the layer of the game class, it would probably be simpler to jus put this into the callback. And please define a unified key combination to lock the camera, one which is probably never used in the game, for example alt + l.